Meta/CMake/all_the_debug_macros.cmake

```diff
@@ -176,6 +176,7 @@ set(LINE_EDITOR_DEBUG ON)
 set(LANGUAGE_SERVER_DEBUG ON)
 set(GL_DEBUG ON)
 set(WASM_BINPARSER_DEBUG ON)
+set(WASM_TRACE_DEBUG ON)
```

Comment:
This change should be part of the previous commit ("LibWasm: Start implementing a naive bytecode interpreter" instead of "LibWasm: Implement most of the remaining instructions")

Reply:
Must've been mixed up in a squash, I'll patch it up